README.md

```diff
@@ -32,7 +32,7 @@ To help with testing app updates before they're released, you can:
 
 ### Contribute Code 
 
-1. Look in [GitHub issues](https://github.com/mattermost/mattermost-server/issues) for issues marked as [Help Wanted]
+1. Look in [GitHub issues](https://github.com/mattermost/mattermost-mobile/issues) for issues marked as [Help Wanted]
```

Comment:
Actually, help wanted issues are listed in the server repo, but they have a `React Native` label. Would you be open to updating the link to `https://github.com/mattermost/mattermost-server/issues?q=label%3A"React+Native"`?

Reply:
Sure thing, amended :)

Reply:
Thanks! Resolving